cpp/dolfinx/common/timing.h

```diff
@@ -27,10 +27,13 @@ Table timings();
 void list_timings(MPI_Comm comm,
                   Table::Reduction reduction = Table::Reduction::max);
 
-/// @brief Return timing (count, total wall time, total user time, total
-/// system time) for given task.
+/// @brief Return timing (count, total wall time) for given task.
 /// @param[in] task Name of a task
 /// @return The (count, total wall time) for the task.
 std::pair<std::size_t, double> timing(std::string task);
 
+/// @brief Logged elapsed times.
+/// @return Elapsed [task id: (count, total wall time)].
+std::map<std::string, std::pair<int, double>> timings();
```

Comment:
There is a discrepancy in the use of `std::size_t` and `int` between `timing` and `timings`.